pyaedt/modules/PostProcessor.py

```diff
@@ -3393,7 +3393,7 @@ def create_fieldplot_cutplane(
             List of cut planes to plot.
         quantity : str
             Name of the quantity to plot.
-        setup_name : str, optional
+        setup : str, optional
             Name of the setup. The default is ``None`` which automatically take ``nominal_adaptive`` setup.
             Please make sure to build a setup string in the form of ``"SetupName : SetupSweep"``
             where ``SetupSweep`` is the Sweep name to use in the export or ``LastAdaptive``.
```

Comment:
```suggestion
            Name of the setup. The default is ``None``, in which case the ``nominal_adaptive`` setup
            is used. Be sure to build a setup string in the form of ``"SetupName : SetupSweep"``,
            where ``SetupSweep`` is the sweep name to use in the export or ``LastAdaptive``.
```
This suggested edits here aren't likely the standard text used elsewhere. All occurrences should be consistent.